src/cpl/share_esmf/laiStreamMod.F90

```diff
@@ -62,6 +62,7 @@ subroutine lai_init(bounds)
     integer                 :: nml_error                  ! namelist i/o error flag
     character(len=CL)       :: stream_fldFileName_lai     ! lai stream filename to read
     character(len=CL)       :: stream_meshfile_lai        ! lai stream meshfile
+    real(r8)                :: lai_dtlimit                ! dlimit for lai stream to use
```

Comment:
This is where you could set the default value to 1.5_r8. This is from the above comments about NOT having a default in the namelist.

Reply:
added `lai_dtlimit = 1.5_r8`